var/spack/repos/builtin/packages/intel-oneapi-mkl/package.py

```diff
@@ -42,10 +45,20 @@ class IntelOneapiMkl(IntelOneApiLibraryPackage):
     def component_dir(self):
         return 'mkl'
 
+    def xlp64_lib(self, lib):
+        return lib + ('_ilp64'
+                      if '+ilp64' in self.spec
+                      else '_lp64')
```

Comment:
(comment - no change necessary) I'm guessing that both of these libraries are installed and this variant helps users select between them (so this could be improved once https://github.com/spack/seps/pull/2 is resolved).

Reply:
> (comment - no change necessary) I'm guessing that both of these libraries are installed and this variant helps users select between them (so this could be improved once [spack/seps#2](https://github.com/spack/seps/pull/2) is resolved).

Yes to everything above.